api/pkg/apps/webapp/cases.go

```diff
@@ -264,13 +264,26 @@ func (h *Server) PostCase(ctx context.Context, id string, w http.ResponseWriter,
 	var kase *cms.Case
 	if id == "" {
 		var err error
+		// record user id of creator if referral
+		// TODO get actual unique identifier rather than name
```

Comment:
It would be nice to create a jira-ticket and the issueKey to the comment (if this is something we want TODO)

Reply:
We can use the `context.Value(ctx, "subject").(string)` to get the user that made that request